notebooks/7-rotation-speeds/interm_rotation_rate.ipynb

```diff
@@ -0,0 +1,2926 @@
+{
```

Comment:
<p>Oh, I had a comment somewhere else about pandas, it gets used here. I think it'd be preferable to use astropy to read the csv as an astropy table; I don't believe we've been using pandas elsewhere, and I don't (yet?) see a need for it as a separate package to import (and figure out).</p><p> </p><p>Also, need data_dir+filename?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>